docs/Documentation/deployment_guide_introduction.md

```diff
@@ -0,0 +1,41 @@
+# OpenVINO™ Deployment {#openvino_docs_deployment_guide_introduction}
+
+@sphinxdirective
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+   Run Inference <openvino_docs_OV_UG_OV_Runtime_User_Guide>
+   Inference Optimization <openvino_docs_deployment_optimization_guide_dldt_optimization_guide>
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+   Deploy Locally <openvino_deployment_guide>
+   Deploy Using Model Server <ovms_what_is_openvino_model_server>
+
+
+Once you have a model that meets both OpenVINO™ and your requirements, you can choose among several ways of deploying it with your application. The two default options are:
+
+.. panels::
+
+    `Deploying locally <openvino_deployment_guide>`_ 
+    ^^^^^^^^^^^^^^
+
+    Local deployment simply uses OpenVINO Runtime installed on the device. It utilizes resources available to the system.
+
+    ---
+
+    `Deploying by Model Serving <ovms_what_is_openvino_model_server>`_
+    ^^^^^^^^^^^^^^
+
+    Deployment via OpenVINO Model Server allows the device to connect to the server set up remotely. This way inference uses external resources instead of the ones provided by the device itself. 
+
+@endsphinxdirective
+
```

Comment:
Just a note: in 2022.3 docs, we also want add documents how to distribute with apt / yum packages. Is it expected we can add one more page reference here and write a dedicated document?

Reply:
We surely can. I'll merge for now and we can add it in a separate PR.